devices/src/virtio/mmio.rs

```diff
@@ -407,6 +454,7 @@ mod tests {
 
     struct DummyDevice {
         acked_features: u32,
+        avail_features: u32,
```

Comment:
You can make this an `u64` to be easier to test with the proper handlers. It was an `u32` to hack a simple test which is no longer required.